commands/Admins/Settings/ownerstats.js

```diff
@@ -0,0 +1,32 @@
+const { Command, version: klasaVersion, Duration } = require('klasa');
+const { version: discordVersion, MessageEmbed } = require('discord.js')
+const { prefix } = require('../../../../config/config.json');
+
+module.exports = class extends Command {
+
+	constructor(...args) {
+		super(...args, {
+			guarded: true,
+			description:'Here, the bot owner can look over his stats, and other things he needs.',
+			permissionLevel: 10,
+			aliases: ['os']
+		});
+	}
+
+	async run(message) {
+		
+	const embed1 = new MessageEmbed()

```

Comment:
If you dont reuse the embed variable multiple times not much sense in making it a variable at all.

Reply:
delete all this